proto/mint/v1beta1/mint.proto

```diff
@@ -81,4 +82,5 @@ message Params {
   int64 minting_rewards_distribution_start_epoch = 7
       [ (gogoproto.moretags) =
             "yaml:\"minting_rewards_distribution_start_epoch\"" ];
+  string strategic_reserve_address = 8 [ (gogoproto.moretags) = "yaml:\"address\"" ];
```

Comment:
leaving this here to prompt discussion - should it be a param or hardcoded? currently a const in mint keeper

Reply:
agreed

Reply:
removed